pkgs/development/tools/misc/doq/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "doq";
+  version = "0.9.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "heavenshell";
+    repo = "py-doq";
+    rev = "refs/tags/${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
I like to specify "refs/tags" because some repositories have branches with names like versions.

Reply:
Ok fair enough.